app/mailers/claim_mailer.rb

```diff
@@ -17,6 +17,11 @@ def rejected(claim)
     view_mail_with_claim_and_subject(claim, "Your claim #{@claim_description} has been rejected, reference number: #{claim.reference}")
   end
 
+  def update_after_three_weeks(claim)
```

Comment:
I guess the one thing we might need to guard against here is if the mailer has been queued to send, but for one reason or another, the delivery has been significantly delayed (for example, maybe Notify has had a catastrophic outage) and the claim has since been approved. Might be worth adding a guard clause to check if the claim has been approved before actually sending the mail.

Reply:
Not a bad shout